packages/autocomplete-core/src/__tests__/getSources.test.ts

```diff
@@ -37,7 +41,98 @@ describe('getSources', () => {
     });
   });
 
-  test.todo('provides default source implementations');
+  test('provides a default source implementations', async () => {
+    const onStateChange = jest.fn();
+    const sources = createSource();
```

Comment:
That's a single source.

```suggestion
    const source = createSource();
```